contrib/test_jenkins.sh

```diff
@@ -139,31 +142,31 @@ if [ -n "$JENKINS_RUN_TESTS" ]; then
         fi
 
         echo Running ucx_perf kit on $hca
-        mpirun -np 2 -mca pml ob1 -mca btl sm,self $AFFINITY $ucx_inst/bin/ucx_perftest -d $hca $opt_perftest
+        mpirun -np 2 -x UCX_ERROR_SIGNALS -x HANDLE_ERRORS -mca pml ob1 -mca btl sm,self $AFFINITY $ucx_inst/bin/ucx_perftest -d $hca $opt_perftest
```

Comment:
It is strange but it show reaction on just `HANDLE_ERRORS`
Also, `UCS_HANDLE_ERRORS` changed.